inc/saihostintf.h

```diff
@@ -861,8 +956,6 @@ typedef struct _sai_hostif_api_t
     sai_remove_hostif_trap_fn                      remove_trap;
     sai_set_hostif_trap_attribute_fn               set_trap_attribute;
     sai_get_hostif_trap_attribute_fn               get_trap_attribute;
-    sai_set_hostif_user_defined_trap_attribute_fn  set_user_defined_trap_attribute;
-    sai_get_hostif_user_defined_trap_attribute_fn  get_user_defined_trap_attribute;
     sai_recv_hostif_packet_fn                      recv_packet;
     sai_send_hostif_packet_fn                      send_packet;
```

Comment:
How to get Switch ID in CB mode ( hif_id == SAI_NULL_OBJECT_ID )?

/**
 * @brief Hostif send function
 *
 * @param[in] hif_id Host interface id. only valid for send through FD channel. Use #SAI_NULL_OBJECT_ID for send through CB channel.
 * @param[in] buffer Packet buffer
 * @param[in] buffer size Packet size in bytes
 * @param[in] attr_count Number of attributes
 * @param[in] attr_list Array of attributes
 *
 * @return #SAI_STATUS_SUCCESS on success Failure status code on error
 */
typedef sai_status_t(*sai_send_hostif_packet_fn)(
        _In_ sai_object_id_t hif_id,
        _In_ void *buffer,
        _In_ sai_size_t buffer_size,
        _In_ uint32_t attr_count,
        _In_ sai_attribute_t *attr_list);